[misc] [refactor] 'ti.vec' and 'ti.veci' is deprecated, directly use a tuple in GUI system instead (#1605)
taichi-dev/taichi: docs/gui.rst

    Draw a line of text on screen.

.. function:: ti.rgb_to_hex(rgb):

>     :parameter rgb: (tuple of 3 floats) The (R, G, B) float values, in range [0, 1)

Reviewer: ```suggestion
    :parameter rgb: (tuple of 3 floats) The (R, G, B) float values, in range [0, 1]
```

>     :return: (RGB hex or np.array of uint32) The translated hex value

Reviewer: ```suggestion
    :return: (RGB hex or np.array of uint32) The converted hex value
```

>     Convert a (R, G, B) tuple of float into a single hex value, e.g.:

Reviewer: ```suggestion
    Convert a (R, G, B) tuple of floats into a single integer value. E.g.,
```

    .. code-block:: python

         rgb = (0.4, 0.8, 1.0)
         hex = ti.rgb_to_hex(rgb)  # 0x66ccff

         rgb = np.array([[0.4, 0.8, 1.0], [0.0, 0.5, 1.0]])
         hex = ti.rgb_to_hex(rgb)  # np.array([0x66ccff, 0x007fff])

>     This is helpful since the return value could be used in other paint APIs.

Reviewer: ```suggestion
    The return values can be used in GUI drawing APIs.
```
